src/app/App.js

```diff
@@ -0,0 +1,34 @@
+import './App.css';
+import { Counter} from "../shared/ui/Counter/Counter";
+import { GenreSelect } from "../shared/ui/GenreSelect/GenreSelect";
+import { SearchForm } from "../shared/ui/SearchForm/SearchForm";
+import {generList} from "../shared/ui/GenreSelect/GenerMockData";
+
+
+function App() {
+  const geners=['Action', 'Comedy', 'Drama', 'Fantasy'];
+
+  const handleSearch = (val) => {
+    console.log(val);
+  };
+
+  const handleChangeSelectedGenre = (val) => {
+    console.log(val);
+  };
+
+  return (
+    <div className="App">
+     <Counter initialValue={5}/>
+     <SearchForm
+     initialValue={""}
+     onSearch={handleSearch}
+     />
+     <GenreSelect 
+      genres={generList}
+      onSelect={handleChangeSelectedGenre}
+     />
+    </div>
+  );
+}
+
+export default App;
```

Comment:
let's get rid of default export by using 

```suggestion
export function App() {
```

Reply:
fixed